README.md

```diff
@@ -2,25 +2,64 @@
 
 # operatorkit
 
-The operatorkit package implements an opinionated framework for developing
-Kubernetes operators. It can be used as a library in golang. It is built on
-top of our [microkit](https://github.com/giantswarm/microkit) framework which
-provides base functionality like logging and error handling.
+The operatorkit is a library for creating [Kubernetes operators][operators]. It
```

Comment:
Is it `the operatorkit`, or just `operatorkit`? I'm thinking it's a proper noun, so it should just be `operatorkit`

Reply:
Done.